docs/pages/admin-guides/access-controls/idps/saml-microsoft-entra-external-id.mdx

```diff
@@ -0,0 +1,235 @@
+
+As a second step of the SAML service provider configuration, you will notice that the Web UI already configures 
+the SAML service provider entity ID, ACS URL and default attribute mapping value that is required for the integration. 
+These values are generated based on preset values and Entra ID `Tenant ID`.
+
+For reference, the values that are auto-configured are listed below:
+
+- Entity ID: `https://login.microsoftonline.com/<entra-id-tenant-id>/`
+- ACS URL: `https://login.microsoftonline.com/login.srf`
+- Attribute:
+  - Name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress`
+  - Value: `uid`
+- Launch URL: `https://portal.azure.com/<entra-id-tenant-id>`. The launch URL is where users are sent to after
+a successful authentication with Teleport.
+
+Provide a display name for this integration (e.g. "azure_portal") and click on button named 
+`Finish` to register service provider. 
+
+## Step 3/5. Create and invite user
+
+In this step, we will create and invite an external user that authenticates with Teleport SAML IdP.
+<Admonition type="info" title="User account domain name">
+The user that is invited should have a matching domain name associated with Teleport SAML IdP. 
+Refer to the Step 1 of this guide.
+</Admonition>
+
+Locate and select **Users** menu under Azure services.
+From the **Users** page, click `+ New User` button and select "Invite external user" option. 
+
+![Invite External User](../../../../img/access-controls/saml-idp/ms-entra-external-id/entra-invite-user.png)
+
+Fill up details for this user as needed to set up the account. You will notice that the a default value for
+"Invite redirect URL" field that points to `https://myapplications.microsoft.com/` is set for this user. 
+This URL is where Microsoft will redirect the user after their first authentication. You can update this URL 
+if you wish to redirect user to a different URL or keep it as it is.
+
+Ensure that the default user type "Guest" remains unchanged. Click the `Review + invite` button to invite this user. 
+
+For this guide, we created a user named `alice@example.com`
+
+As an invited user, the first step is to redeem the invitation. They can do so by clicking on 
+invitation link that can be found in the invitation email. As part of the redemption process, 
+Microsoft asks users to confirm their consent to share user details with the IdP.
+
+![User consent](../../../../img/access-controls/saml-idp/ms-entra-external-id/entra-user-consent.png)
+
+Users must accept the consent in order to enable authentication via Teleport. 
+
+
+## Step 4/5. Grant Azure roles to the user
+
+By default, there aren't any permissions granted to the invited users. The general process to grant a role is 
+to create a [Azure Resource Group](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal#what-is-a-resource-group)
```

Comment:
```suggestion
to create an Azure resource group
```

As part of the style guide, we discourage inline links in how-to guides to keep the reader's focus on the instructions. In this case, I think we can remove the link since a resource group is a core Azure concept (it's also not capitalized, as far as I can see in the Azure docs).